### ISSUE TITLE
Add internationalisation for China

### ISSUE THREAD

**Issue by jgadsden** [OWASP/threat-dragon#350]
**Describe what problem your feature request solves**
We have various language translations available for version 2.0 Threat Dragon. It would be good to have a translation for China as well

**Describe the solution you'd like**
Provide the translation, similar to the other translations under `td.vue/src/i18n/*.js`, and then fix up `td.vue/src/i18n/index.js` to read something like:

```
import Vue from 'vue';
import VueI18n from 'vue-i18n';

import cn from './cn.js';
import el from './el.js';
import en from './en.js';
import es from './es.js';
import fr from './fr.js';
import pt from './pt.js';

Vue.use(VueI18n);

let i18n = null;

const get = () => {
    if (i18n === null) {
        i18n = new VueI18n({
            locale: 'en',
            messages: {
                cn,
                el,
                en,
                es,
                fr,
                pt
            }
        });
    }
    return i18n;
};

export const tc = (key) => get().tc(key);

export default {
    get
};
```

**Additional context**
A similar process was for a translation to Spanish in #319

**Comment by jgadsden** [comment]
@taoyiyang1 is this something you could take on?

**Comment by taoyiyang1** [comment]
> @taoyiyang1这是你可以接受的吗？

@jgadsden Of course, I think it's great

**Comment by jgadsden** [comment]
Thanks @taoyiyang1 , much appreciated

**Comment by taoyiyang1** [comment]
hello @jgadsden 
I don’t seem to be able to create a branch, which branch should I operate on

**Comment by jgadsden** [comment]
That is odd - it must be to do with github roles. This repo is part of the OWASP github organisation and so the permissions are role based. I can look into this, but to get it going I have created branch 'v2/i18n-cn' for you @taoyiyang1

**Comment by taoyiyang1** [comment]
> 这很奇怪 - 这一定与 github 角色有关。这个 repo 是 OWASP github 组织的一部分，所以权限是基于角色的。我可以调查一下，但为了让它继续下去，我为你创建了分支“v2/i18n-cn” @taoyiyang1

ok , thank

**Comment by taoyiyang1** [comment]
@jgadsden  
The "v2/i18n-cn" branch does not seem to be created based on the v2/development branch

**Comment by jgadsden** [comment]
Oops, sorry @taoyiyang1 , can you try it now?

**Comment by taoyiyang1** [comment]
> 哎呀，对不起@taoyiyang1，你现在可以试试吗？
@jgadsden  No problem

**Comment by jgadsden** [comment]
@taoyiyang1 has merged PR  #351 which satisfies this issue
